packages/open-payments/src/client/token.ts

```diff
@@ -0,0 +1,45 @@
+import { HttpMethod, ResponseValidator } from 'openapi'
+import { RouteDeps } from '.'
+import { getASPath, AccessToken } from '../types'
+import { post } from './requests'
+
+interface RotateRequestArgs {
+  url: string
+}
+
+export interface TokenRoutes {
+  rotate(args: RotateRequestArgs): Promise<AccessToken>
+}
+
+export const rotateToken = async (
+  deps: RouteDeps,
+  args: RotateRequestArgs,
+  validateOpenApiResponse: ResponseValidator<AccessToken>
+) => {
+  const { axiosInstance, logger } = deps
+  const { url } = args
+
+  return post(
+    {
+      axiosInstance,
+      logger
+    },
+    {
+      url
+    },
```

Comment:
Based on the auth Open Payments spec we also need to pass in `accessToken` I believe